MMOCoreORB/src/server/zone/objects/creature/commands/ResourceCommand.h

```diff
@@ -56,6 +56,9 @@ class ResourceCommand : public QueueCommand {
 			} else if(command == "create") {
 				giveResource(creature, &args);
 
+			} else if(command == "ghdump") {
```

Comment:
Why don't we perform the galaxyharvester dump at the same time as the swgcraft and regular resource dump with `/resource dump`?

The writer format is different with XML and organized by planet, so we'll need to group things in a vector by zone instead of writing it out immediately, but I think that would make things simpler for a server management point-of-view and for maintenance.

Pseudo code:

```
Vector<String, String> ghSpawnStrings;
for(int i = 0; i < resourceMap->size(); i++) {
  ManagedReference<ResourceSpawn*> spawn = resourceMap->get(i);
  
  // normal dump stuff
  // swgcraft dump stuff
  // build the gh string and add it to the vector by spawn name
}

for(int i = 0; j < planets->size(); i++) {
  ZoneResourceMap* zoneMap = resourceMap->getZoneResourceList(planets->get(j));
  for (int j = 0; j < ghSpawnStrings.size(); j++) {
    if (zoneMap->containsSpawn(ghSpawnStrings.get(j)) {
      // write out <Planet> with this spawn string
    }
  }
}
```